sdk/formrecognizer/Azure.AI.FormRecognizer/src/CopyModelOperation.cs

```diff
@@ -45,7 +45,18 @@ public class CopyModelOperation : Operation<CustomFormModelInfo>
         public override string Id { get; }
 
         /// <inheritdoc/>
-        public override CustomFormModelInfo Value => OperationHelpers.GetValue(ref _value);
+        public override CustomFormModelInfo Value
+        {
+            get
+            {
+                if (HasCompleted && !HasValue)
+#pragma warning disable CA1065 // Do not raise exceptions in unexpected locations
+                    throw new InvalidOperationException("Copy model operation failed");
```

Comment:
We don't do this in Java.
Is it something only applicable for .NET or should other languages consider to follow?